pom.xml

```diff
@@ -161,26 +194,6 @@
 
   <build>
     <plugins>
-      <!-- jacoco code coverage -->
-      <plugin>
-        <groupId>org.jacoco</groupId>
-        <artifactId>jacoco-maven-plugin</artifactId>
-        <version>0.8.7</version>
-        <executions>
-          <execution>
-            <goals>
-              <goal>prepare-agent</goal>
-            </goals>
-          </execution>
-          <execution>
-            <id>report</id>
-            <phase>test</phase>
-            <goals>
-              <goal>report</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
```

Comment:
Do we still need the `<build>` tag? The parent pom is for the all the module's bom, which *shouldn't* have any tests to run? iirc, you already moved the build configs to the jar-parent.

Reply:
Good point. I'll remove.